src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role role);
+
+    Role getByRoleName(Role.RoleName roleName);

```

Comment:
according to the requirements of the task, the input parameter must be of type String
![image](https://user-images.githubusercontent.com/55763108/236458161-5cbc4acb-e79e-426f-9017-2160fa16b200.png)